static/bidder-params/openweb.json

```diff
@@ -2,25 +2,19 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "OpenWeb Adapter Params",
   "description": "A schema which validates params accepted by the OpenWeb adapter",
-
   "type": "object",
   "properties": {
     "placementId": {
-      "type": "integer",
+      "type": "string",
       "description": "An ID which identifies this placement of the impression"
     },
-    "siteId": {
-      "type": "integer",
-      "description": "An ID which identifies the site selling the impression"
-    },
-    "aid": {
-      "type": "integer",
-      "description": "An ID which identifies the channel"
-    },
-    "bidFloor": {
-      "type": "number",
-      "description": "BidFloor, US Dollars"
+    "org": {
+      "type": "string",
+      "description": "The organization ID.",
+      "minLength": 1
     }
   },
-  "required": ["aid"]
+  "required": [
+    "org"
+  ]
```

Comment:
@IgorKulemzin  this might be an breaking change for publisher who are still including `aid` and `org` as bidder param in imp object

requesting to consider providing some transition time so that publisher has enought time to migrate from  `aid` to `org`